source/landing-zone.txt

```diff
@@ -9,9 +9,252 @@ Landing Zone Design
+
+Consider the following organization-specific
+requirements when you create a landing zone:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Consideration
+     - Description
+   
+   * - System Partition Requirements
+     - Identify how you will partition systems for management. For
+       example, clarify how your team should arrange {+service+}
+       projects or {+service+} organizations. 
+       
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-hierarchy`.
+   * - Access Controls
+     - Identify MongoDB Atlas Control Plane access controls, and
+       database access controls for both workload and workforce principals. Create a comprehensive list of principals and mechanisms for how you will authenticate and authorize. Define Atlas API key access controls, including authorizations and expiration rules.
+   * - Change Control and Auditability Requirements
+     - Clarify any change control or audit controls requirements. This
+       can include change approval processes and tools, along with
+       reporting guidelines.
+
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-auditing-logging`.
+   * - Data Residency Requirements
+     - Identify data residency requirements, such as data sovereignty
+       requirements, or application data partitioning rules.
+   * - Network Topology Requirements
+     - Identify network topology requirements, including Cloud Provider
+       regions, private connectivity options, and application
+       deployment topology.
+
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-network-security`.
+   * - High Availability Requirements
+     - Identify high availability requirements such as cluster topology
+       with node count and priority per region, global write
+       requirements, and partitioning tolerances.
+
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-high-availability`.
+   * - Data Retention Requirements
+     - Identify and record your data retention policies. This may
+       require creating a classification for automation, including
+       creation of archive and/or purge automation. In some cases, data
+       must be preserved for a certain duration, whereas in other cases
+       data must be purged after some duration. Identify performance
+       characteristics of retrieval of archived records.
+   * - Disaster Recovery Requirements
+     - Identify all disaster recovery requirements, including:
+
+       - Identify and record your current disaster recovery plan. 
+       - Define backup snapshot schedules and retentions.
+       - Identify Restore Time Objective (RTO) and Restore Point
+         Objective (RPO). 
+       - Define backup snapshot locations. 
+       - Identify Point-in-time restore option selection. 
+       - Define recovery methodologies (i.e., restore, queryable
```

Comment:
Our [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/general-term-guidelines/avoid-obscure-words/) discourages this too.
```suggestion
       - Define recovery methodologies (that is, restore, queryable
```